src/platforms/common/profiling/index.mdx

```diff
@@ -159,6 +169,35 @@ The <PlatformIdentifier name="profiles_sample_rate" /> setting is *relative* to
 
 </PlatformSection>
 
+
+<PlatformSection supported={["rust"]}>
```

Comment:
This section is missing a code snippet for enabling performance for Rust. See PlatformSections for other supported SDKs above.